client/src/cmdlfhitag.c

```diff
@@ -278,7 +278,7 @@ static int CmdLFHitagEload(const char *Cmd) {
     if (dumplen == 48 ||  dumplen == 4 * 64) {
 
         lf_hitag_t *payload =  calloc(1, sizeof(lf_hitag_t) + dumplen);
-        
+
```

Comment:
looks like you somehow didn't base your PR on latest source and got unrelated stuff in your PR